### PR TITLE
6.single review

### PR DESCRIPTION
**Pull request by mini-wizbit**
NOTE! Accidently made a branch on a branch....

### CODE REVIEW

File: nc-games/src/components/ReviewById/ReviewById.jsx [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+            <li>
+                <h2>{reviewNumber.title}</h2>
+                <img src={reviewNumber.review_img_url} alt={reviewNumber.title} className="reviewById-card-Img"></img>
+                <p>{reviewNumber.review_body}</p>
+                <h4>{reviewNumber.votes}</h4>
+                <h4>{reviewNumber.owner}</h4>
+                <p>{reviewNumber.category}</p>
+            </li>
+
+        </div>
```
Comment (douglashellowell) [inline nc-games/src/components/ReviewById/ReviewById.jsx]: why are you using `<li>` here?

File: nc-games/src/components/ReviewById/ReviewById.jsx [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+    const [reviewNumber, setReviewNumber]= useState(1)
+
+
+    useEffect(()=>{
+        getReviewById(review_id)
+        .then((thisReview)=>{
+            setReviewNumber(thisReview)
+        })
+    },[review_id]) 
+
```
Comment (douglashellowell) [inline nc-games/src/components/ReviewById/ReviewById.jsx]: i've quite confused about `reviewNumber` here. It sounds like a number - and starts as `1` - but you set it to the review itself later on!

Would be more readable if you kept the data types more consistent (and maybe rename the state if it helps)

**Comment by douglashellowell** [comment]
good stuff!

----

One point from a previous PR - you should make use of the `params` object to handle queries in an `axios` request!

No need to handle it yourself with if statements

---

Symantically a `<ul>`s direct children should be `<li>`s
In `AllReviews.jsx` you have this sort of structure

```html
<ul>
      <div>
            <div>
                   <li> </li>
            <div>
            <div>
                   <li> </li>
            <div>
      <div>
</ul>
```
> where it should be more like 
```html
<ul>
           <li> </li>
           <li> </li>
</ul>
```
If you need the divs for styling maybe put them _inside_ instead

**Comment by douglashellowell** [comment]
It looks like `getReviewsByCategories` and `getReviews` both do the same thing?

----


Be sure to implement a Loading pattern or you users will be staring at a blank screen on slower connections!


----

Please put in another `/nchelp pr` when you've added some loading patterns

File: nc-games/src/components/ReviewById/ReviewById.jsx (hunk 1 above)
Comment (mini-wizbit) [reply nc-games/src/components/ReviewById/ReviewById.jsx]: Good question.... habit??

**Comment by douglashellowell** [comment]
Some fixes done but some things I mentioned aren't fixed!

* remove the strange `<li>` in `ReviewById.jsx`
* refactor&rename `reviewNumber` state in `ReviewById.jsx`
* refactor the `<div>` in `Allreviews.jsx` on line 28


Could you make sure to format your code with prettier before you commit again?
You may need to `right click > format document with... > configure default formatter...`

**Comment by douglashellowell** [comment]
(also i was talking aboutget `getReviewsByCategories` and `getReviews` - not `getCategories`. )

You're currently not using `getReviewsByCategory` - you could delete it if it's redundant now!

**Review by hevdevs — CHANGES_REQUESTED** [review]
Looks good based on previous comments however we still aren't using the axios params object in our get request for getReviews - this is worth looking into now as later we'll also want to sort reviews by different properties (using Google and the docs to look up axios request options should help here)

File: nc-games/src/utils/api.js [hunk 3]
```diff
@@ -4,12 +4,28 @@ const gamesApi = axios.create({
...
+      return data;
+    });
+  }
+  // i feel like this is just more code??
+  /*let queryStr = "/reviews";
   if (category) queryStr += `?category=${category}`;
   return gamesApi.get(queryStr).then(({ data }) => {
     return data;
-  });
+  });*/
```
Comment (pkaodev) [inline nc-games/src/utils/api.js]: remove unused code

**Comment by pkaodev** [comment]
merge away

**Pull request merged**